src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -50,8 +51,42 @@ class DetectedLanguageSuitev4 extends TestBase with DataFrameEquality with TextK
     assert(iso(0).get(0).toString == "en" && iso(0).get(1).toString == "es" &&
       iso(1).get(0).toString == "fr" && iso(1).get(1).toString == "zh")
   }
+  test("Language Detection - Check Confidence Score") {
+    val replies = detector.transform(df)
+      .select("output", "output.result.confidenceScore")
+      .collect()
+    assert(replies(0).schema(0).name == "output", "output.result.confidenceScore")
+      val firstRow = replies(0)
+      val secondRow = replies(1)
+    assert(replies(0).schema(0).name == "output")
+    val fromRow = DetectLanguageResponseV4.makeFromRowConverter
+    replies.foreach(row => {
+        val resFirstRow = fromRow(firstRow.getAs[GenericRowWithSchema]("output"))
+        val resSecondRow = fromRow(secondRow.getAs[GenericRowWithSchema]("output"))
+        val modelVersionFirstRow = resFirstRow.modelVersion.get
+        val modelVersionSecondRow = resSecondRow.modelVersion.get
+        val positiveScoreFirstRow = resFirstRow.result.head.get.confidenceScore
+        val negativeScoreFirstRow = resFirstRow.result.head.get.confidenceScore
+        val neutralScoreFirstRow = resFirstRow.result.head.get.confidenceScore
+        assert(modelVersionFirstRow.matches("\\d{4}-\\d{2}-\\d{2}"))
+        assert(modelVersionFirstRow.matches("\\d{4}-\\d{2}-\\d{2}"))
+        assert(positiveScoreFirstRow > 0.5)

```

Comment:
for each sentence, just need to check one score to be larger than 0.5. If positive, check positive score. If negative, check negative score.